src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -88,38 +90,30 @@ public Uri CreateServiceInsightUri(StoredMessage message)
 
         public SagaData GetSagaById(Guid sagaId) => GetModel<SagaData>(new RestRequestWithCache(string.Format(SagaEndpoint, sagaId), RestRequestWithCache.CacheStyle.IfNotModified)) ?? new SagaData();
 
-        public PagedResult<StoredMessage> Search(string searchQuery, int pageIndex = 1, string orderBy = null, bool ascending = false)
+        public PagedResult<StoredMessage> GetAuditMessages(Endpoint endpoint, string searchQuery = null, string orderBy = null, bool ascending = false)
         {
-            var request = CreateMessagesRequest();
+            var request = CreateMessagesRequest(endpoint?.Name);
```

Comment:
This is now the only call to the `CreateMessagesRequest` method, so the parameter should be made mandatory there.